cpp/include/cudf/detail/iterator.cuh

```diff
@@ -313,6 +315,74 @@ struct scalar_pair_accessor : public scalar_value_accessor<Element> {
   }
 };
 
+/**
+ * @brief Utility to discard template type arguments.
+ *
+ * Substitute for std::void_t.
+ *
+ * @tparam T Ignored template parameter
+ */
+template <typename... T>
+using void_t = void;
+
+/**
+ * @brief Compile-time reflection to check if `Element` type has a `rep()` member.
+ */
+template <typename Element, typename = void>
+struct has_rep_member {
+  static constexpr bool value = false;
+};
```

Comment:
```suggestion
template <typename Element, typename = void>
struct has_rep_member : std::false_type {};
```

Reply:
Thanks. TIL.

Reply:
Thanks for the suggestion. This is cleaner.